### ISSUE TITLE
what is wrong with my syntax?

### ISSUE THREAD

**Issue by EvanDawesome** [creationix/js-git#117]
if (response === "cool stuff") {
	confirm ("You think your cooler than me huh?");
}
var cooler = prompt("ansar meh yes or noo dameet!")
if (cooler === "no") {
	confirm ("Dan I truly em betta den yue");
}
else if (cooler === "yes" || cooler === "") {
	confirm ("Oh yeee? Den let us cee who as beetar at maty!");
	confirm ("We do hmmm. Leet us say five to feeth poower. GO!");
}
babyAnswers = [3125,3120,3130]
var babyFinal = (babyAnswers[Math.floor(Math.random() * 3)])
playerAnswer = prompt("Teel meh ansar!")
if (playerAnswer === 3125 && babyFinal === 3125) {
	confirm ("I guess we booth get it coect.")
}
else if (playerAnswer !== 3125 && babyFinal === 3125) {
	confirm ("I am bettar dan yue sook my donkey!")
}
else if (playerAnswer === 3125 && babyFinal !== 3125) {
	confirm ("You won ya soon of goon!")
}
else if (playerAnswer !== 3125 && babyFinal !== 3125) {
	confirm ("We are sooch fudging god dam idioots!");
}

so basically if you respond no to the prompt cooler it still shows the confirm ("We are sooch fudging god dam idioots!"); and the prompt ("Teel meh ansar!") can anyone help? Also if you say yes and give the correct answer for playerAnswer it either says the last if else or the second one.
